scanpipe/tests/test_models.py

```diff
@@ -243,6 +248,7 @@ def test_scanpipe_run_model_get_run_id_method(self):
 
     def test_scanpipe_run_model_queryset_methods(self):
         now = timezone.now()
+        Run.objects.all().delete()
```

Comment:
Is this because tests are not independent?

Reply:
The Run QS should be scoped to the project instead, this may be cleaner.
I'll commit a fix before the merge.